tests/phpunit/quiz_settings_test.php

```diff
@@ -167,6 +168,39 @@ public function test_browser_exam_keys_validation_errors($bek, $expectederrorstr
         $this->assertContains($expectederrorstring, $errors);
     }
 
+    public function test_validate_sebconfigfile_success() {
+        $user = $this->getDataGenerator()->create_user();
+        $this->setUser($user);
+        $xml = "<?xml version=\"1.0\" encoding=\"UTF-8\"?>\n"
+            . "<!DOCTYPE plist PUBLIC \"-//Apple//DTD PLIST 1.0//EN\" \"http://www.apple.com/DTDs/PropertyList-1.0.dtd\">\n"
+            . "<plist version=\"1.0\"><dict><key>hashedQuitPassword</key><string>hashedpassword</string>"
+            . "<key>allowWlan</key><false/></dict></plist>\n";
+        $itemid = $this->create_test_file($xml);
+        $quizsettings = new quiz_settings(0, (object) [
+            'quizid' => 1,
+            'requiresafeexambrowser' => settings_provider::USE_SEB_UPLOAD_CONFIG,
+            'sebconfigfile' => $itemid,
+        ]);
+        $quizsettings->save();
+        $errors = $quizsettings->get_errors();
+        $this->assertEmpty($errors);
+    }
+
+    public function test_validate_sebconfigfile_failure() {
```

Comment:
Nice that you added negative scenario test as well :+1:

Reply:
Thats all @andrewmadden  :tada: